analyzer-java/src/test/java/com/infosupport/ldoc/analyzerj/AnalysisVisitorTest.java

```diff
@@ -249,7 +249,7 @@ void comment_tests() {
         parse("""
             class Example {
               /**
-               * This method is an example. These are the remarks.
```

Comment:
With this change, this comment is no longer a Javadoc comment.

For example:

```java
/**
 * Hello <remarks>world.
 */
public class Echo<T> {
  void echo() {
  }
}
```

Running the javadoc tool (Tools > Generate JavaDoc in IDEA) now shows "1 error" and exits 1:

```
/home/wander/Code/echo/src/main/java/Echo.java:2: error: unknown tag: remarks
 * Hello <remarks>world.
         ^
1 error

"javadoc" finished with exit code 1
```

Basically, I don't think we should invent our own syntax here. Existing Java projects will have existing Javadoc documentation comments which they expect to remain compatible with the javadoc tool. I do not expect they will change to some other format just for this.

I already linked this, but the format that the Javadoc tool expects is described here, which is where I took the "first sentence is summary" convention from: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#firstsentence

Reply:
I see your point in that case I will just revert it to the original way and fix the bug.